build.fsx

```diff
@@ -101,59 +56,53 @@ Target "CleanDocs" (fun _ ->
 // Build library & test project
 
 Target "Build" (fun _ ->
-    !! solutionFile
-    |> MSBuildRelease "" "Rebuild"
-    |> ignore
+    DotNetCli.Build (fun p ->
+        { p with
+            Project = solutionFile
+            Configuration = "Release" })
 )
 
 // --------------------------------------------------------------------------------------
 // Run the unit tests using test runner
 
 Target "RunTests" (fun _ ->
-    !! testAssemblies
-    |> NUnit (fun p ->
-        { p with
-            DisableShadowCopy = true
-            TimeOut = TimeSpan.FromMinutes 20.
-            OutputFile = "TestResults.xml" })
-)
-
-#if MONO
-#else
-// --------------------------------------------------------------------------------------
-// SourceLink allows Source Indexing on the PDB generated by the compiler, this allows
-// the ability to step through the source code of external libraries https://github.com/ctaggart/SourceLink
-
-Target "SourceLink" (fun _ ->
-    let baseUrl = sprintf "%s/%s/{0}/%%var2%%" gitRaw (project.ToLower())
-    use repo = new GitRepo(__SOURCE_DIRECTORY__)
-    !! "src/**/*.fsproj"
-    |> Seq.iter (fun f ->
-        let proj = VsProj.LoadRelease f
-        logfn "source linking %s" proj.OutputFilePdb
-        let files = proj.Compiles -- "**/AssemblyInfo.fs"
-        repo.VerifyChecksums files
-        proj.VerifyPdbChecksums files
-        proj.CreateSrcSrv baseUrl repo.Revision (repo.Paths files)
-        Pdbstr.exec proj.OutputFilePdb proj.OutputFilePdbSrcSrv
-    )
+    try
+        DotNetCli.Test(fun p ->
+            { p with
+                Project = "tests/FSharp.Control.AsyncSeq.Tests"
+                Configuration = "Release"
+                TimeOut = TimeSpan.FromMinutes 20. })
+    finally
+        AppVeyor.UploadTestResultsXml AppVeyor.TestResultsType.NUnit "bin"
 )
-#endif
 
 // --------------------------------------------------------------------------------------
 // Build a NuGet package
 
 Target "NuGet" (fun _ ->
-    Paket.Pack(fun p -> 
+    (*
+    Paket.Pack(fun p ->
          { p with
             Version = release.NugetVersion
-            ReleaseNotes = toLines release.Notes})
+            ReleaseNotes = toLines release.Notes })
+    *)
+    DotNetCli.Pack (fun p ->
+        { p with
+            Project = project
+            Configuration = "Release"
+            VersionSuffix = buildVersion
+            OutputPath = buildDir
+            AdditionalArgs =
+              [ "--no-build"
+                //"/p:ReleaseNotes=" + (toLines release.Notes)
```

Comment:
you should use property `Version`, like `/p:Version=12.3` to pass whole version.
the `VersionSuffix` is bad, because incomplete and doesnt work as expected (because compatibility)

```fsharp
sprintf "/p:Version=%s" release.NugetVersion
```

Reply:
Thanks for the great feedback! By the way, should the `/p:ReleaseNotes` that I commented out work correctly?